enumerables.rb

```diff
@@ -0,0 +1,155 @@
+module Enumerable
+    def my_each
+        for i in 0...self.length do
+            yield(self[i])
+        end
+    end
+    
+    def my_each_with_index
+        for i in 0...self.length do
+            yield(self[i], i)
+        end
+    end
+
```

Comment:
- [ ] enumerables #my_each_with_index returns an enumerator if no block is given

- [ ] enumerable #my_each_with_index when a block is given when self is an array returns the array. Again always convert the yield result to an array then return self.
Hint: try and use `size.times do || ` instead of `for i in 0....self.length do`